src/gui/sdlmain.cpp

```diff
@@ -2165,19 +2165,30 @@ Bitu GFX_SetSize(int width, int height, const Bitu flags, double scalex,
 
 		int is_framebuffer_srgb_capable = 0;
 		if (SDL_GL_GetAttribute(SDL_GL_FRAMEBUFFER_SRGB_CAPABLE,
-		                        &is_framebuffer_srgb_capable))
+		                        &is_framebuffer_srgb_capable)) {
 			LOG_WARNING("OPENGL: Failed getting the framebuffer's sRGB status: %s",
 			            SDL_GetError());
+		}
 
-		sdl.opengl.framebuffer_is_srgb_encoded = RENDER_UseSRGBFramebuffer() && is_framebuffer_srgb_capable > 0;
+		sdl.opengl.framebuffer_is_srgb_encoded =
+		        RENDER_UseSrgbFramebuffer() &&
+		        (is_framebuffer_srgb_capable > 0);
 
-		if (RENDER_UseSRGBFramebuffer() && !sdl.opengl.framebuffer_is_srgb_encoded)
+		if (RENDER_UseSrgbFramebuffer() &&
+		    !sdl.opengl.framebuffer_is_srgb_encoded) {
 			LOG_WARNING("OPENGL: sRGB framebuffer not supported");
+		}
 
-		// Using GL_SRGB8_ALPHA8 because GL_SRGB8 doesn't work properly with Mesa drivers on certain integrated Intel GPUs
-		const auto texformat = RENDER_UseSRGBTexture() && sdl.opengl.framebuffer_is_srgb_encoded
+		// Using GL_SRGB8_ALPHA8 because GL_SRGB8 doesn't work properly
+		// with Mesa drivers on certain integrated Intel GPUs
+		const auto texformat = RENDER_UseSrgbTexture() && sdl.opengl.framebuffer_is_srgb_encoded
 		                             ? GL_SRGB8_ALPHA8
 		                             : GL_RGB8;
+
+		// if (texformat == GL_SRGB8_ALPHA8) {
+		// 	LOG_MSG("OPENGL: Using sRGB texture");
+		// }
```

Comment:
I've never looked at the render code in DosBox so I'm probably not the best person for review.  But I don't like commented out code like this.  I'd say either wrap this in an `#ifdef` for debug builds or just remove it.

Reply:
They definitely won't be removed as they are super useful for focused debugging sessions. Writing them over and over again is a pain. Also , I don't like using "debug" log levels because then you get flooded with all sorts of debug messages you don't care about when enabling debug (or "trace") logging.

Another non-solution is logging categories -- too complicated to maintain, and at the end of the day, often I only want one of two specific debug messages and that's it...

What could be done, perhaps, is surrounding them with `#if 0` / `#endif`, so you can toggle them on by changing 0 to 1. But... dunno, I don't have problems with the current comment based solution. It does the job with minimal hassle.

@FeralChild64 @shermp @kklobe your views?

Reply:
Btw, these were trivial one-liners. What I wrote is more about "heavy-duty" logging like this:

https://github.com/dosbox-staging/dosbox-staging/blob/main/src/capture/image/image_scaler.cpp#L139-L189

(Although, maybe a bit of a non-example as I wrapped this in a function, so I can easily only toggle calling this function... 😄)

Reply:
Just an idea that I've seen SDL code do is wrap these in (for example) `#ifdef DEBUG_RENDERER `

Then when you're working on this specific part of the code, you can just type `#define DEBUG_RENDERER` at the top of the file (or use the build system to pass the flag in) and you get everything turned on.

But if you just want to stick to commented out code, I won't make too big of a fuss over it.  I just think it looks kind of ugly.

Reply:
I'm personally happy to switch to `#if 0` highlighting; like I said, I want to be able to toggle comments on an individual basis. Toggling them wholesale with `DEBUG_RENDERER` won't cut it as I explained. Too noisy, they most often need to be toggled individually in most debugging tasks.

So for example this

<img width="699" alt="image" src="https://github.com/dosbox-staging/dosbox-staging/assets/698770/1d554b22-d6c1-404a-b30d-c1226cf57f39">

Would become this

<img width="687" alt="image" src="https://github.com/dosbox-staging/dosbox-staging/assets/698770/f17121d4-5c09-484c-a161-2591726664eb">

So unless objections from others, I'm happy to start using this style going forward. I was concerned that I won't see these blocks highlighted as comments, but the Vim C++ syntax plugin does support `#if 0`/`#if 1` highlighting out of the box, so that's great 😄

Reply:
And as I said, for multi-line debug log statements changing `0` to `1` and back is actually a lot easier than toggling both ends of the comment block, or per line (if you editor doesn't support commenting/uncommenting whole blocks with one shortcut), so it was a good suggestion @weirddan455 👍🏻